src/main/java/taxi/service/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package taxi.service;
+
+import java.util.Optional;
+import taxi.dao.DriverDao;
+import taxi.exception.AuthenticationException;
+import taxi.lib.Inject;
+import taxi.lib.Service;
+import taxi.model.Driver;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Inject
+    private DriverDao driverDao;
```

Comment:
Perhaps, you don't need use dao here, it'll be better use driverService. because this service has method for getting driver by login.